torch/utils/data/datapipes/utils/common.py

```diff
@@ -24,7 +24,8 @@ def get_file_pathnames_from_root(
         root: str,
         masks: Union[str, List[str]],
         recursive: bool = False,
-        abspath: bool = False) -> Iterable[str]:
+        abspath: bool = False,
+        deterministic_order: bool = True) -> Iterable[str]:
```

Comment:
Super nit: Could you please make the default value as `False` to align with other default values like`non_deterministic`?

Reply:
To be clear, I think @ejguan means to change `deterministic_order: bool = True` to `non_deterministic: bool = False` just so that it is consistent with other patterns we have in the codebase.

After that you can have:
```python
if not non_deterministic:
    files.sort()
    dirs.sort()
```

Reply:
Changed. Just in case, note that sorting `dirs` if it's `not recursive` is wasteful, and that the `files` sorting needs to go before the `for`.